Lib/test/test_syntax.py

```diff
@@ -229,7 +229,7 @@
 
 >>> with a as b
 Traceback (most recent call last):
-SyntaxError: invalid syntax
+SyntaxError: expected one of: ':', ','
```

Comment:
The list of tokens is very punctuation heavy. Maybe don't use a comma, just a space? So
```
SyntaxError: expected one of: ':' ','
```
Even that is rather jittery -- would just using spaces them work?
```
SyntaxError: expected one of : ,

Reply:
I tried originally the first form but we changed it to the current one based on consistency with other errors but we can certainly change it back. 



> Even that is rather jittery -- would just using spaces them work?

I think it may be confused with actual punctuation like saying "expected one of A, B". Maybe not a lot in this case but when other tokens appear it may be confusing, although it looks a bit cleaner.

What about surrounding them in some form of brackets likeL

`expecting one of: [':',  ',']`

Reply:
Hm, brackets are even more jittery. I'm guessing just a space between the string literals instead of a comma would be great.

Reply:
I have removed the comma in my last commit